petpals/templates/forum.html

```diff
@@ -138,37 +129,37 @@ <h1>Forum</h1>
 
             <!-- Forum List -->
             <div class="forum-main-body p-2 p-sm-3 collapse forum-content show">
-              <!--First Entry-->
+              {% for post, user in posts %}
               <div class="forum-card mb-2">
                 <div class="forum-card-body p-2 p-sm-3">
                   <div class="media forum-item">
                     <a
-                      href="{{ url_for('profile_router.profile_user', username='1') }}"
+                      href="/user/{{ post.author.username }}"
                       data-toggle="collapse"
                       data-target=".forum-content"
                       ><img
-                        src="https://bootdey.com/img/Content/avatar/avatar1.png"
+                        src="{{ user.image_path }}"
```

Comment:
Should be `post.author.image_path`. Sorry if there was any confusion from my previous review.